PusherServer/BatchEvent.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace PusherServer
+{
+    /// <summary>
+    /// Represents an event that is part of a batch trigger
+    /// </summary>
+    public class BatchEvent
+    {
+        /// <summary>
+        /// The name of the event
+        /// </summary>
+        public string name { get; set; }
+
+        /// <summary>
+        /// The event data
+        /// </summary>
+        public string data { get; set; }
```

Comment:
Both will fall foul of the DefaultJSONSerialiser: capital property names (I imagine) will become capital JSON field names, which are not valid. An object will be serialised in-line, not treated as opaque data.